tutorials/napps/how_to_protect_a_napp_rest_endpoint.rst

```diff
@@ -0,0 +1,313 @@
+:tocdepth: 2
+:orphan:
+
+.. _tutorial-how-to-protect-a-napp-rest-endpoint:
+
+###################################
+How to create your own NApp: Part 4
+###################################
+
+********
+Overview
+********
+
+In this tutorial, you will learn how to create a Network Application (NApp) for
+*Kytos* (|kytos|_) to communicate with your application. This NApp will be made
+with a public and a private REST endpoints.
+
+The average time to go through this is: ``20 min``.
+
+What you will learn
+===================
+
+* How to create a superuser;
+* How to create a NApp with a public and a private REST endpoint.
+
+What you will need
+==================
+
+* How to create a basic NApp: Part 1 - Refer to |Tutorial_01|_;
+* How to create a basic NApp: Part 2 - Refer to |Tutorial_02|_;
+* How to create a basic NApp: Part 3 - Refer to |Tutorial_03|_;
+* Read the documentation: |protect_rest|_;
+* Your |dev_env|_ already up and running;
+* The Storehouse NApp, installed and running.
+
+
+************
+Introduction
+************
+
+Besides events, another method to communicate with your NApp is
+through REST endpoints. These endpoints can be public or private
+(please note that the latter is still under development).
+
+The creation of a new endpoint is made by the use of the `@rest` decorator,
+and it can be optionally protected using the `@authenticated`
+decorator. More information about this module can be found in
+the |protect_rest|_ and in the Auth documentation.
```

Comment:
Is there another page with the Auth documentation whose link is missing from here? Please add link.